grid.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import Ogre
+
+class GridFloor:
+    def __init__(self, scene, material_name):
```

Comment:
it does not make sense to have material_name configurable, as the grid has some very specific requirements on what it should be